readme.md

```diff
@@ -22,59 +22,71 @@ Hello, Node!
 
 ### Instructions & Guiding Questions
 
-- [ ] Begin by renaming `data/pets.sample.json` to `pets.json`
+- [x] Begin by renaming `data/pets.sample.json` to `pets.json`
 
 * **Question:** In this lesson we will be modifying the contents of the `pets.json` file. Why do you think that file appears inside of the `.gitignore` file?
 
-* **Your Answer:**
+* **Your Answer:** We don't want git to track the changes to this file.
 
 ---
 
-- [ ] Take a look at the NodeJS documentation for the [fs module](https://nodejs.org/api/fs.html). When looking at documentation like this it can be overwhelming to start but you likely know more than you think. Take a moment to find a few concepts you understand.
+- [x] Take a look at the NodeJS documentation for the [fs module](https://nodejs.org/api/fs.html). When looking at documentation like this it can be overwhelming to start but you likely know more than you think. Take a moment to find a few concepts you understand.
 
 ---
 
-- [ ] One of the most common patterns we'll come across as web developers is the concept of [CRUD](https://www.codecademy.com/articles/what-is-crud). Take a moment to define what CRUD represents.
+- [x] One of the most common patterns we'll come across as web developers is the concept of [CRUD](https://www.codecademy.com/articles/what-is-crud). Take a moment to define what CRUD represents.
 
 * **Question:** Imagine you have a file called `classmates.txt`. For each part of CRUD, describe how the action would interact with the file.
 
 * **Your Answer:**
+C: Create the file or an entry in the file
+R: Read or view an entry from the file
+U: Update an entry in the file
+D: Delete someone from the file (or the wholefile)
+
 
 ---
 
-- [ ] Consider the above and then look back through the [fs module](https://nodejs.org/api/fs.html) documentation.
+- [x] Consider the above and then look back through the [fs module](https://nodejs.org/api/fs.html) documentation.
 
 * **Question:** What methods represent each CRUD action?
 
 * **Your Answer:**
-
+C: fs.write()
+R: fs.read()
+U: fs.appendFile()
+D: fs.ulink()
 ---
 
-- [ ] Take a look at the following two methods: [fs.readFile()](https://nodejs.org/api/fs.html#fs_fs_readfile_path_options_callback) and [fs.readFileSync()](https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options).
+- [x] Take a look at the following two methods: [fs.readFile()](https://nodejs.org/api/fs.html#fs_fs_readfile_path_options_callback) and [fs.readFileSync()](https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options).
 
 * **Question:** What is the difference between these two methods?
 
 * **Your Answer:**
+fs.readFileSync() is asynchronous where fs.readFile() is not. fs.readFile() has a callback function.
```

Comment:
Other way around! `fs.readFileSync()` is synchronous, which means it will block other processes on the server to make sure it finishes. `fs.readFile()` is asynchronous.